youtubefunction.py

```diff
@@ -6,8 +6,9 @@
 
 class YoutubeFunction :
     #This function returns a list of youtube video URLs
-    def getYouTube(self, queryy) :
+    def getYouTube(self, q) :
         
+            queryy = q.replace(" ","+")
             url = ('https://www.youtube.com/results?search_query=' +str(queryy) + '&sp=CAMSAhAB')

```

Comment:
Just saw `&sp=CAMSAhAB` - can you add a comment that talks about why you have this extra URL field?